frontend/src/cljs/imas_seamap/map/views.cljs

```diff
@@ -352,10 +352,10 @@
         {:position "bottomright"
          :style nil}]
 
-       (when has-info?
+       (when (and has-info? (not= fi-status :feature-info/empty))
```

Comment:
I don't recall details, but can you both `has-info?` and have a status of `:feature-info/empty`?